src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h

```diff
@@ -233,7 +233,7 @@ namespace OpenMS
       OPENMS_PRECONDITION(transition_group.chromatogramIdsMatch(), "Chromatogram native IDs need to match keys in transition group")
 
       MRMFeature mrmFeature;
-      mrmFeature.setIntensity(0.0);
+      mrmFeature.setIntensity();
```

Comment:
```suggestion
      mrmFeature.setIntensity(0.0);
```

Reply:
also the accumulate changes will not compile